cmd/root.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	config "github.com/kyallanum/athena/v1.0.0/models/config"
+	library "github.com/kyallanum/athena/v1.0.0/models/library"
+	logs "github.com/kyallanum/athena/v1.0.0/models/logs"
+	"github.com/spf13/cobra"
+)
+
+var configFile, logFile string
+
+var rootCmd = &cobra.Command{
+	Use:           "athena [flags]",
+	Short:         "A text and log file parser to discern important information",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(configFile) == 0 {
+			return fmt.Errorf(`required flag(s) "config" not set`)
+		}
+		if len(logFile) == 0 {
+			return fmt.Errorf(`required flag(s) "log-file" not set`)
+		}
+		return nil
+	},
+}
+
+func Execute(logger *log.Logger) error {
```

Comment:
I know I added a logger in my code-cleanup method. This was me messing around with it and I'm not quite ready to implement that. Let's revert everything back to fmt and I'll work on switching to a logger when I have the chance.

Reply:
Noted.

Reply:
@kyallanum  Also, look into [logrus](https://github.com/sirupsen/logrus). It's a great tool that streamlines logging.